linux_os/guide/system/software/sudo/sudoers_no_root/oval/shared.xml

```diff
@@ -0,0 +1,48 @@
+
+<def-group>
+   <definition class="compliance" id="sudoers_no_root" version="1">
+     {{{ oval_metadata("YOUR SHORT DESCRIPTION HERE") }}}
+     <criteria operator="AND">
+        <criterion comment="Make sure that no rules in sudoers reference root or ALL" test_ref="test_no_roo_or_ALL_rules" />
+        <criterion comment="Make sure that rules in sudoers reference users" test_ref="test_no_user_spec_rules" />
+     </criteria>
+  </definition>
+
+  <ind:textfilecontent54_test check="all" check_existence="none_exist"
+  comment="Make sure that no rules in sudoers reference root or ALL"
```

Comment:
Suggestion for more specific comment:
`Make sure that no rules in sudoers define a Runas_Spec for root or ALL`